tests/ce/test_ce.py

```diff
@@ -0,0 +1,33 @@
+from openfisca_us_data import CE
+
+
+def test_ce_from_2019():
+    """Test the Consumer Expenditure data generating functions for 2019"""
+    ce = CE()
+    ce.generate(2019)
+    ce_2019 = ce.load(2019)
+
+    assert len(set(ce_2019.keys()).intersection({"annual", "household"})) == 2
+
+    # Test household average demographic estimate -------------------------
+    est_income_bf_tax = ce_2019["/annual/income_before_tax"][()]
+    assert round(est_income_bf_tax) == 82743
```

Comment:
could you make 82743 a constant and refer to that? same with others here

Reply:
done